morpheus/_lib/include/morpheus/stages/preallocate.hpp

```diff
@@ -71,12 +94,38 @@ struct PreallocateStageInterfaceProxy
         std::vector<std::tuple<std::string, TypeId>> needed_columns);
 };
 
-// Explicit instantiations
-template class PreallocateStage<MessageMeta>;
-template class PreallocateStage<MultiMessage>;
```

Comment:
Why did you remove the explicit instantiation here? Can we use this on more types?

Reply:
I can probably roll this back; at the time I started updating things there was a bug here that was causing linking errors; I think David has since fixed it.